crypto/rand/rand_lib.c

```diff
@@ -113,7 +216,24 @@ void rand_cleanup_int(void)
     CRYPTO_THREAD_lock_free(rand_engine_lock);
 #endif
     CRYPTO_THREAD_lock_free(rand_meth_lock);
-    rand_drbg_cleanup();
+    CRYPTO_THREAD_lock_free(rand_bytes.lock);
+    OPENSSL_clear_free(rand_drbg.randomness, rand_drbg.size);
```

Comment:
Should `rand_bytes.buff` be cleared too?
Should `RAND_DRBG_uninstantiate` clear `rand_drbg.randomness`?